algebird-core/src/main/scala/com/twitter/algebird/MomentsGroup.scala

```diff
@@ -53,7 +53,51 @@ sealed class Moments(val m0D: Double, val m1: Double, val m2: Double, val m3: Do
     math.sqrt(m0D) * m3 / math.pow(m2, 1.5)
```

Comment:
I wonder if this is slower than `math.sqrt(m0D) * m3 / (m2 * math.sqrt(m2))`

or even: `m3 / (m2 * stddev)`

seems like maybe sqrt and div are about the same:

https://stackoverflow.com/questions/6884359/c-practical-computational-complexity-of-cmath-sqrt

so I guess the `m3 / (m2 * stddev)` formula would be best because it is clearest. What do you think?

Reply:
I agree, that's definitely clearest!